website/docs/r/kubernetes_fleet_update_run.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "Container"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_kubernetes_fleet_update_run"
+description: |-
+  Manages a Kubernetes Fleet Update Run.
+---
+
+# azurerm_kubernetes_fleet_update_run
+
+Manages a Kubernetes Fleet Update Run.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "westeurope"
+}
+
+resource "azurerm_kubernetes_fleet_manager" "example" {
+  location            = azurerm_resource_group.example.location
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  hub_profile {
+    dns_prefix = "example-dns-prefix"
+  }
+}
+
+resource "azurerm_kubernetes_fleet_update_strategy" "example" {
+  name                        = "example"
+  kubernetes_fleet_manager_id = azurerm_kubernetes_fleet_manager.example.id
+  stage {
+    name = "example-stage-1"
+    group {
+      name = "example-group-1"
+    }
+    after_stage_wait_in_seconds = 21
+  }
+}
+
+resource "azurerm_kubernetes_fleet_update_run" "example" {
+  name                        = "example"
+  kubernetes_fleet_manager_id = azurerm_kubernetes_fleet_manager.example.id
+  managed_cluster_update {
+    upgrade {
+      type               = "Full"
+      kubernetes_version = "1.27"
+    }
+    node_image_selection {
+      type = "Latest"
+    }
+  }
+  fleet_update_strategy_id = azurerm_kubernetes_fleet_update_strategy.example.id
+}
```

Comment:
Should this also have the `stage` be included in the `ignore_changes`?

Reply:
Thanks for the suggestion, I've updated it to use a more recommended example.